src/VirtoCommerce.Platform.Core/Common/SemanticVersion.cs

```diff
@@ -137,13 +137,21 @@ public override bool Equals(object obj)
 
         public override int GetHashCode()
         {
-            int result = _version.GetHashCode();
+            var result = _version.GetHashCode();
             result = result * 31 + Prerelease.GetHashCode();
             return result;
         }
 
         #region IComparable Members
 
+        public static bool operator >=(SemanticVersion a, SemanticVersion b) => a.CompareTo(b) >= 0;
```

Comment:
Seems it can be NullReference when a = null. Also is think (null, null) case should be handled.
So probably need to remove these from here, or to implement this logic.

Reply:
Also need to add tests for these operators, if we add them.

Reply:
Added null check, added tests